docs/snippets/ov_auto_batching.py

```diff
@@ -1,41 +1,33 @@
-#include <openvino/runtime/core.hpp>
+from openvino.runtime import Core
 
-int main() {
-    ov::Core core;
-    auto model = core.read_model("sample.xml");
+core = Core()
+model = core.read_model(model="sample.xml")
 
-//! [compile_model]
-{
-    auto compiled_model = core.compile_model(model, "GPU", ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT));
-}
-//! [compile_model]
+# [compile_model]
+config = {"PERFORMANCE_HINT": "THROUGHPUT"}
+compiled_model = core.compile_model(model, "GPU", config)
```

Comment:
why config is not passed as `core.compile_model(model, "GPU", config={"PERFORMANCE_HINT": "THROUGHPUT"})`?

Reply:
Just my personal preference but I am gonna change it for your approach

Reply:
ok, we can use different approaches in different examples to demonstrate all the variants.